src/plugins/tables/reports/a11y-1-en.hbs

```diff
@@ -0,0 +1,15 @@
+---
+{
+	"title": "Accessibility pre-assessment #1 - Tables",
+	"language": "en",
+	"description": "Partial. Pre-evaluation of Success Criteria at Level A and Level AA of WCAG 2.1",
+	"tag": "tables",
+	"parentdir": "tables",
+	"parentPage": "Tables",
+	"parentPageURL": "table",
```

Comment:
The parent page filename do include a "s" before the language suffix. see https://github.com/wet-boew/wet-boew/blob/master/src/plugins/tables/tables-en.hbs 

```suggestion
	"parentPageURL": "tables",
```

Reply:
fixed